code/modules/multiz/movement.dm

```diff
@@ -186,11 +186,23 @@
 	if(..())
 		return species.can_fall(src)
 
+/atom/movable/proc/protected_from_fall_damage(var/turf/landing)
+	return !!(locate(/obj/structure/stairs) in src)
+
+/mob/protected_from_fall_damage(var/turf/landing)
+	. = ..()
+	if(!.)
+		// This is very silly, but it can be refined and made more appropriate as our multiz turf system is expanded.
+		var/obj/item/storage/backpack/parachute/parachute = get_equipped_item(slot_back_str)
+		if(istype(parachute) && parachute.packed)
+			parachute.packed = FALSE
```

Comment:
Since this will by definition never fire twice in a landing, I think it's probably safe to have a `visible_message` here to signal that the parachute has deployed.